about.html

```diff
@@ -0,0 +1,141 @@
+        <li class="top-section-list"><img src="Assets\icons8-facebook-18.png" alt="facebook"/></li>
+        <li class="top-section-list"><img src="Assets\icons8-twitter-18.png" alt="twitter"/></li>
+        <li class="top-section-list">English</li>
+        <li class="top-section-list">My Page</li>
+        <li class="top-section-list">Logout</li>
+      </ul>
+     </div>
+ 
+     <!-- Navbar -->
+     <!--Mobile-->
+     <nav class="navbar-mobile">
+       <div class="menu">
+         <a> <img src="Assets\mobile_menu 1.png" class="menubar" alt="menu"> </a>
+          </div>
+     </nav>
+
+     <!-- Nav Items -->
+     <nav class="navbar-container">
+      <a href="index.html" class="p-logo"><p class="logo">WIT</p></a>
+      <ul class="navbar-items">
+        <li>
+          <a href="about.html" class="menu-link-desktop">About</a>
+        </li>
+
+        <li>
+          <a href="#" class="menu-link-desktop">Program</a>
+        </li>
+
+        <li>
+          <a href="#" class="menu-link-desktop">Join</a>
+        </li>
+
+        <li>
+          <a href="#" class="menu-link-desktop">Sponsor</a>
+        </li>
+
+        <li>
+          <a href="#" class="menu-link-desktop">News</a>
+        </li>
+
+        <li class="list-items">
+          <a href="#">WITMP 2023</a>
+        </li>
+      </ul>
+    </nav>
+</header>
+<!-- Header Ends Here -->
+
+<!-- Main Starts Here -->
+<section class="about-theme-info">
+<!-- Hero -->
+<div class="about-first-section">
+<div>
+<h4 class="about-theme-intro">"Join us this year!"</h4>
+<h2 class="about-theme">WOMEN IN TECH <br> MENTORSHIP PROGRAMME <br> 2023.</h2>
+</div>
+
+<div class="about-intro">
+<h4 class="about-intro-text">
+Are you a woman trying to navigate the tech space? This mentorship program for women in tech contribute to the retention and advancement of female talent, as mentees gain confidence, expand their professional networks, and acquire the necessary skills to excel in their careers, leading to increased diversity and innovation in the tech sector. <span class="button-span">Click the button below to register now.</span>
+</h4>
+<a href="#" class="button-a">
+  <button class="button"> Click Here to Register  </button></a>
+</div>
+</div>
+
+<div class="about-second-section">
+<div class="about-program">
+  <h4 class="about-main-program">See past Women In Tech mentorship program</h4>
+  <hr class="about-one">
+  <h5 class="about-summit">Take a look at the past three Women In Tech mentorship program which took place in Abuja, in Lagos and in Enugu.</h5>
+</div>
+
+<div class="about-program-container">
+
+  <div class="about-program-details">
+    <h5 class="about-detail-name">2020</h5>
+    <p class="about-detail-text">Women In Tech Annual Mentorship program in Abuja.</p>
+  </div>
+
+  <div class="about-program-details1">
+    <h5 class="about-detail-name">2021</h5>
+    <p class="about-detail-text">Women In Tech Annual Mentorship program in Lagos.</p>
+  </div>
+
```

Comment:
- [ ] please kindly use two cards (images) as well in the mobile version of the about page.
- [ ] the logo show be on the left and text on the right on the footer of the mobile about
- [ ] please, the middle section and logo should be added as well.
yours
![Screenshot (1169)](https://github.com/JoyTaribagshaw/module-1-capstone-project/assets/65298048/02d6765a-e3b0-41ee-b043-a006fee296cb)
original
![Screenshot (1168)](https://github.com/JoyTaribagshaw/module-1-capstone-project/assets/65298048/d304e7c9-eef6-4db2-a484-489f0b4b23d6)

Reply:
Thank you so much for your review, I have fixed the cards.

![mobile-cards](https://github.com/JoyTaribagshaw/module-1-capstone-project/assets/111312788/db1f0081-2136-4bae-8fcc-517e34bbf420)